variables.tf

```diff
@@ -39,10 +39,16 @@ variable "ingress_annotations" {
   default     = {}
 }
 
-variable "manifests" {
-  description = "Path/URL to manifests to be applied after argocd is deployed"
-  default     = []
-  type        = list(string)
+# variable "manifests" {
+#   description = "Path/URL to manifests to be applied after argocd is deployed"
+#   default     = []
+#   type        = list(string)
+# }
+
+variable "manifests_directory" {
+  description = "Path/URL to directory that contains manifest files to be applied after argocd is deployed"
```

Comment:
Update description to `Path to directory`. Remove the URL. This is because `kubectl_path_documents` only supports paths